docs/sites.yml

```diff
@@ -2293,3 +2293,22 @@
   built_by: Peter Kroyer
   built_by_url: https://www.peterkroyer.at/
   featured: false
+- title: Geddski
+  main_url: https://gedd.ski
+  url: https://gedd.ski
+  description: >
+    Frontend mastery blog - level up your UI game.
+  categories:
+    - Web Dev
+    - CSS
+    - JavaScript
+    - Grid
+    - Flexbox
+    - PWA
+    - Productivity
+    - UI
+    - UX
+  built_by: Dave Geddes
+  built_by_url: https://twitter.com/geddski
+  # you know you want to...
```

Comment:
😆

Reply:
😁

Reply:
yep totally gonna optimize it with those image plugins. Gatsby is so dope.

Reply:
Heh, this is fun, but could you remove that comment? https://next.gatsbyjs.org/docs/site-showcase-submissions/#note-on-featured-sites :)

Reply:
done :D